envoy-control-tests/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/config/EnvoyControlTestConfiguration.kt

```diff
@@ -31,6 +32,7 @@ object RandomConfigFile :
 
 abstract class EnvoyControlTestConfiguration : BaseEnvoyTest() {
     companion object {
+        private val logger by logger()
```

Comment:
This seems to be unused

Reply:
Forgot to add logs that I wanted. Will fix it :)